### PR TITLE
Replace CPython default branch name to main branch.

### PR DESCRIPTION
**Pull request by Mariatta**
Do not merge until CPython branch has actually been renamed, after the 3.10 beta 1 has been released.
Currently scheduled for Monday, 2021-05-03

https://mail.python.org/archives/list/python-committers@python.org/message/QWW7KGBW5UH2N5FOZOFXQBQPYELWQM3O/

### CODE REVIEW

**Comment by tiran** [comment]
Thanks!

Could you please include documentation how to change a local checkout to use ``main``? This question is going to come up a lot.

**Comment by Mariatta** [comment]
Thanks. I've added the info here: https://github.com/python/devguide/pull/662/commits/7c7751183d1e39005999b1fd6e6ef80d23ec647b

File: gitbootcamp.rst [hunk 1]
```diff
@@ -153,6 +153,19 @@ To delete a **remote** branch::
 
 You may specify more than one branch for deletion.
 
+
+Renaming Branch
```
Comment (Mariatta) [inline gitbootcamp.rst:157]: Added the instruction on how to rename local branch here.

**Comment by tiran** [comment]
Fantastic! Thanks for your speedy response.

**Comment by Mariatta** [comment]
@pablogsal Please merge after the 3.10 release. Thanks.

**Comment by pablogsal** [comment]
:rocket:  Thanks @Mariatta :heart:

**Pull request merged**